File: auth/src/main/java/com/diagorn/sparkathon/auth/dto/auth/LogoutResponse.java
package com.diagorn.sparkathon.auth.dto.auth;

import io.swagger.v3.oas.annotations.media.Schema;
import lombok.Builder;
import lombok.Data;
import lombok.ToString;

/**
 * Logout response
 *
 * @author diagorn
 */
@Data
@Builder
@ToString
public class LogoutResponse {
    /**
     * Logout succeeded
     */
    @Schema(description = "Logout succeeded", example = "true")
    private boolean success;
    /**
     * Comments
     */
    @Schema(description = "Comments", example = "Success")
    private String message;
}

File: auth/src/main/java/com/diagorn/sparkathon/auth/dto/auth/LogoutResponse.kt
package com.diagorn.sparkathon.auth.dto.auth

import io.swagger.v3.oas.annotations.media.Schema

/**
 * Logout response
 *
 * @author diagorn
 */
class LogoutResponse(
    /**
     * Logout succeeded
     */
    @Schema(description = "Logout succeeded", example = "true")
    val success: Boolean,

    /**
     * Comments
     */
    @Schema(description = "Comments", example = "Success")
    val message: String
)